src/TLuaInterpreter.cpp

```diff
@@ -12494,7 +12494,14 @@ void TLuaInterpreter::logError( std::string & e, const QString & name, const QSt
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(green, black, s2 );
         mpHost->mpEditorDialog->mpErrorConsole->printDebug(red, black, s3 );
     }
-
+    
+    if( mpHost->mpConsole && mpHost->mEchoLuaErrors )
```

Comment:
Similar to keneanung, I will have the editor open and prefer to have it echo'd to the main window regardless. I keep the editor window open 100% of the time I use Mudlet, usually on another monitor, so I think an always-on option is optimal for my purposes at least.